src/components/Home/Shreyansh/index.tsx

```diff
@@ -15,6 +15,14 @@ const Shreyansh: React.FC = () => {
             Week-4
           </button>
         </Link>
+        <Link to='week5'>
+          <button
+            className={`weekButton ${selectedWeek === 'week5' ? 'active' : ''}`}
+            value='week5'
+            onClick={(e) => setSelectedWeek(e.currentTarget.value)}>
+            Week-5
+          </button>
+        </Link>
       </div>
       <Outlet />
     </ShreyanshWrapper>
```

Comment:
It is been asked to remove this name from the css file.Dont use person's name in main component file